Metadata/Driver/AnnotationDriver.php

```diff
@@ -72,6 +72,10 @@ public function loadMetadataForClass(\ReflectionClass $class)
 
         foreach ($this->reader->getClassAnnotations($class) as $annot) {
             if ($annot instanceof Service) {
+                if (!isset($annot->decorationInnerName) && isset($annot->decoration_inner_name)) {
```

Comment:
Are you referring to https://github.com/schmittjoh/JMSDiExtraBundle/pull/255/files#diff-795eb25e349a10927c913d862c5cf87aR83 ?

Because they are two different deprecation notices... This checks for the annotation usage. The other one checks for the use in MetaData. It is still useful if someone is loading Metadata by a mean other than Annotations.

I will still provide a new version with the `null ===` tests